test/files/neg/for-comprehension-val.check

```diff
@@ -1,25 +1,26 @@
-for-comprehension-val.scala:5: error: `val` keyword in for comprehension is unsupported: instead, bind the value without `val`
-  for (x <- 1 to 5 ; val y = x) yield x+y       // fail
-                           ^
```

Comment:
treat `val x <-` too

Reply:
that was previously a syntax error. The midstream assignment was deprecated since 2.10, it should have been made an error in 2.11.